prime-router/src/testIntegration/kotlin/transport/AS2TransportIntegrationTests.kt

```diff
@@ -113,7 +113,7 @@ class AS2TransportIntegrationTests {
         val header = makeHeader()
         setupLogger()
         every { as2Transport.sendViaAS2(any(), any(), any(), any(), any()) }
-            .throws(WrappedAS2Exception("cannot connect", ConnectException()))
+            .throws(WrappedAS2Exception.wrap(ConnectException()))
```

Comment:
`WrappedAS2Exception` constructor became a static method in the new version of the library.